grafana-plugin/src/containers/OutgoingWebhook2Form/OutgoingWebhook2Form.tsx

```diff
@@ -49,35 +72,167 @@ const OutgoingWebhook2Form = observer((props: OutgoingWebhook2FormProps) => {
     [id]
   );
 
+  if (
+    (action === WebhookFormActionType.EDIT_SETTINGS || action === WebhookFormActionType.VIEW_LAST_RUN) &&
+    !outgoingWebhook2Store.items[id]
+  ) {
+    // nothing to show if we open invalid ID for edit/last_run
```

Comment:
can it be unloaded in store yet at the moment this code is running?

Reply:
I'm not sure what you mean by this. This is when you're hitting an ID that was previously deleted or it just doesn't exist. There's no such object in the store.

Reply:
as I understand we don't show anything if there is no such item in the store, but in this case we need to force load it within this component